node-operators/networks/telemetry.md

```diff
@@ -200,7 +253,3 @@ sudo systemctl start gantree-relay && journalctl -f -u gantree-relay
 ```
 
 You should see waiting for provisioning in the logs.  Once it's complete, you can log into the [https://app.gantree.io](https://app.gantree.io) and select networks. You will see a `View Monitoring Dashboard` link to your custom Prometheus / Grafana dashboard which you can customize to your needs.  
```

Comment:
Gantree waits at "waiting for provisioning" until they log back into the portal and click "provision dashboard".  then it switches to "provisioning" which can take a few minutes.  they don't always see this, but they do the first time.